index.js

```diff
@@ -1,17 +1,235 @@
+const { google } = require('googleapis');
 const fs = require('fs');
-const path = require('path');
+const has = require('lodash.has');
+const { klona } = require('klona');
 
 module.exports = {
-  bundle: {
-    directory: 'modules',
-    modules: getBundleModuleNames()
+  improve: '@apostrophecms/form',
+  fields: {
+    add: {
+      googleSheetSubmissions: {
+        label: 'aposForm:activateGoogleSubmit',
+        type: 'boolean'
+      },
+      googleSpreadsheetId: {
+        label: 'aposForm:googleSheetId',
+        type: 'string',
+        required: true,
+        htmlHelp: 'aposForm:googleSheetIdHtmlHelp',
+        if: {
+          googleSheetSubmissions: true
+        }
+      },
+      googleSheetName: {
+        label: 'aposForm:googleSheetName',
+        type: 'string',
+        help: 'aposForm:googleSheetNameHelp',
+        if: {
+          googleSheetSubmissions: true
+        }
+      }
+    },
+    group: {
+      afterSubmit: {
+        fields: [
```

Comment:
I left the label out here, which means no label on current main branch, but I'll have a PR that maintains the group label when group fields are combined.